contributors/devel/cherry-picks.md

```diff
@@ -19,26 +19,46 @@ task is for backporting PRs from master to release branches.
    github.com/github/hub` assuming you have a standard golang development
    environment.
 
-## Initiate a Cherry Pick
- * Run the [cherry pick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh).
+## Initiate a Cherry-pick
+ * Run the [cherry-pick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh).
    This example applies a master branch PR #98765 to the remote branch
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
- * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   * Be aware the cherry-pick script assumes you have a git remote called `upstream` that points at the Kubernetes github org.
+   Please see our [recommended Git workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage).
+   * You will need to run the cherry-pick script separately for each patch release you want to cherry-pick to.
+
+ * Your cherry-pick PR targeted to the release branch will immediately get the
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestones on cherry-pick PRs should be the milestone for the target release branch (for example, milestone 1.11 for a cherry-pick onto release-1.11).
+   * The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager) for the target release branch is responsible for
+   applying the `cherry-pick-approved` label.
+   You may cc them with `<@githubusername>` on your cherry-pick PR.
+
+## Cherry-pick Review
+
+Cherry-pick pull requests have an additional requirement compared to normal pull
+requests.
+They must be approved specifically for cherry-pick by Approvers.
+Additionally, the [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
+is the final and sole authority on removing the `do-not-merge/cherry-pick-not-approved`
+label and triggering a merge into the current patch release.
+
+## Searching for Cherry-picks
+
+[A sample search on kubernetes/kubernetes pull requests that are labeled as `cherry-pick-approved`](https://git.k8s.io/kubernetes/pulls?q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved)
+[A sample search on kubernetes/kubernetes pull requests that are labeled as `do-not-merge/cherry-pick-not-approved`](https://github.com/kubernetes/kubernetes/pulls?q=is%3Aopen+is%3Apr+label%3Ado-not-merge%2Fcherry-pick-not-approved)
```

Comment:
I think this needs a bullet point.

Reply:
Yep. This needs bullet points.
Also, the first query should be: `https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved`